### PR TITLE
Wallet: Services added

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-wallet/src/services/Services.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+interface Config {
+  relayerUrl: string;
+  jsonRpcUrl: string;
+}
+
+export interface Services {
+  sdk: any;
+}
+
+const Services = (config: Config) => {
```
Comment (sz-piotr) [inline universal-login-wallet/src/services/Services.ts]: This function being uppercase is misleading. If I were using it I would assume it's a class. You can change it to `createServices` **unless** it is the same in other projects - consistency is more important.

File: universal-login-wallet/src/services/Services.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const UniversalLoginSDK = require('universal-login-sdk').default;
```
Comment (sz-piotr) [inline universal-login-wallet/src/services/Services.ts]: `import UniversalLoginSDK from 'universal-login-sdk';`

File: universal-login-wallet/test/Services.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import {expect} from 'chai';
+import createServices from '../src/services/Services';
+
+describe('Services', () => {
+  it('Should create services', () => {
+    const services = createServices({relayerUrl: 'http://relayer.com', jsonRpcUrl: 'http://mockprovider.com'});
+    expect(services).to.not.be.null;
+    expect(services.sdk).to.not.be.null;
+  });
+});
```
Comment (sz-piotr) [inline universal-login-wallet/test/Services.ts]: Doesn't the linter require newlines?

Comment (JustynaBroniszewska) [reply universal-login-wallet/test/Services.ts]: Now it does 😃

**Pull request merged**